apps/fluent-tester/package.json

```diff
@@ -101,32 +102,22 @@
     "flow-bin": "^0.113.0",
     "metro-config": "^0.66.2",
```

Comment:
I think metro needs to be bumped to 0.67 per https://reactnative.dev/blog/2022/03/30/version-068

Reply:
Hmm, rnxx-kit bug?

Reply:
Not a bug. This gets resolved to the correct version when adding "metro-config" and "metro-react-native-babel-preset" to the "capabilities" section in the rnx-kit section below. Just didn't realize I needed to add them :)

Reply:
Actually, it seems that it won't automatically update "metro-react-native-babel-preset". Not sure why but I'll manually update it.